src/datasource.js

```diff
@@ -156,25 +82,79 @@ export class GenericDatasource {
     });
   }
 
-  selectQueryFunc() {
+  suggestTags(type, key) {
+    if (!key) {
+      // Need at least some characters typed in order to suggest something
+      return this.q.when([]);
+    }
     return this.backendSrv.datasourceRequest({
-      url: this.url + '/status',
+      url: this.url + '/' + type + 's/tags/' + key + ':*',
       method: 'GET',
-      headers: {'Content-Type': 'application/json'}
-    }).then(response => {
-      var version = response.data['Implementation-Version'];
-      var regExp = new RegExp('([0-9]+)\.([0-9]+)\.(.+)');
-      if (version.match(regExp)) {
-        var versionInfo = regExp.exec(version);
-        var major = versionInfo[1];
-        var minor = versionInfo[2];
-        if (major == 0) {
-          if (minor < 17) {
-            return this.getDataLegacy;
+      headers: this.createHeaders()
+    }).then(result => {
+      if (result.data.hasOwnProperty(key)) {
+        return [' *'].concat(result.data[key]).map(value => {
+          return {text: value, value: value};
+        });
+      }
+      return [];
+    });
+  }
+
+  metricFindQuery(query) {
```

Comment:
Yes, called by grafana itself to get variable values. But not documented anywhere :disappointed: